app/code/core/Mage/Adminhtml/Block/Widget/Grid/Advanced/Abstract.php

```diff
@@ -0,0 +1,100 @@
+<?php
+/**
+ * OpenMage
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Adminhtml
+ * @copyright  Copyright (c) 2006-2020 Magento, Inc. (https://www.magento.com)
+ * @copyright  Copyright (c) 2018-2023 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+use Mage_Adminhtml_Block_Widget_Grid_Advanced_Abstract as AdvancedGrid;
+
+/**
+ * Grid widget massaction block
+ *
+ * @category   Mage
+ * @package    Mage_Adminhtml
+ *
+ */
+abstract class Mage_Adminhtml_Block_Widget_Grid_Advanced_Abstract extends Mage_Adminhtml_Block_Widget
+{
+    /**
+     * Sets Advanced Gird template
+     */
+    public function __construct()
+    {
+        parent::__construct();
+        $this->setResetConfirmText(Mage::helper('core')->jsQuoteEscape(Mage::helper('core')->__('Are you sure?')));
```

Comment:
@empiricompany I've changed this text, the previous one was slightly wrong ("resetted" is a typo actually) but most of all this way translations are already in every language pack